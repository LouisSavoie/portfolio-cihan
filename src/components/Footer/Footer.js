import React from 'react'
import { Container } from 'react-bootstrap'

const Footer = () => {
  return (
    <footer data-testid="footer">
      <Container className="p-3 text-center">
        <span className="p-auto">Â© 2022-present. All rights Reserved.</span>
      </Container>
    </footer>
  )
}

export default Footer
