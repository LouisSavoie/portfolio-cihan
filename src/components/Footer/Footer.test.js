import React from 'react'
import { render, screen, cleanup } from '@testing-library/react'
import '@testing-library/jest-dom'
import renderer from 'react-test-renderer'
import Footer from './Footer'

afterEach(() => {
  cleanup()
})

test('should render Footer', () => {
  render(<Footer />)
  const FooterElement = screen.getByTestId('footer')
  expect(FooterElement).toBeInTheDocument()
})

test('matches snapshot', () => {
  const tree = renderer.create(<Footer />).toJSON()
  expect(tree).toMatchSnapshot()
})
