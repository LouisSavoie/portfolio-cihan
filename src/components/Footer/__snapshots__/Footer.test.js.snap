// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<footer
  className="mt-3"
  data-testid="footer"
>
  <div
    className="p-3 text-center container"
  >
    <span
      className="p-auto"
    >
      Â© 2022-present. All rights Reserved.
    </span>
  </div>
</footer>
`;
